### DISCUSSION TITLE
Set Discord verification level to "High" instead of "Highest"

### DISCUSSION THREAD

**Discussion by ghost** [mastercoms/tf2cfg#483] [General]
All legitimate users have an Email address, but not all have a phone number. After being changed to Highest, I can no longer chat. Additionally people can still join the server and DM others anyways, even if the verification is set to Highest, defeating the purpose of the change.

**Comment by melvyn2** [comment]
Until recently I did not either have a phone number discord deemed acceptable, so I don't believe that enforcing phone number verification is worthwhile.

**Comment by micwoj92** [comment]
I was stupid enough to give my phone number to Discord about 2 years ago, do not make this mistake kids. Use mastercomfig issues like @TheThunderGuyS did to communicate with us, discord phone number giving peasants...

**Comment by mastercoms** [comment]
Many large servers and other non-Discord communities have this requirement, and I will be keeping this for now. If you have any concerns, you can continue to share them during this trial period, but this verification level will be set for the time being.

It is unfortunate that some legitimate users will be blocked from the Discord community with this level, but I hold security to the highest priority.

**Comment by mastercoms** [comment]
After some monitoring of the situation and some extra consideration, I have changed the verification level back to "Medium".
